sys/ztimer/auto_init.c

```diff
@@ -109,7 +106,7 @@
 #    if ZTIMER_RTT_FREQ != FREQ_1KHZ
 #      define ZTIMER_MSEC_CONVERT_LOWER_FREQ ZTIMER_RTT_FREQ
 #    endif
-#  else
+#  elif defined(ZTIMER_TIMER)
```

Comment:
I think there should be an `#else` clause added that gives an `#error` in cause `ztimer_msec` is used, but neither the `periph_rtt` nor the `periph_timer` backend is selected. This might not be possible to configure, but better be safe than sorry.

Reply:
error messages for no Backend selected (e.g.: cause there is non) happen in step 3